ui/ace/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@craftercms/ace-adapter",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Roy Art",
+  "license": "LGPL-3.0-only",
+  "private": true,
+  "dependencies": {
+    "ace-builds": "^1.4.12"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "build": "chmod +x build.sh; ./build.sh"
```

Comment:
@sumerjabri @jdrossl please review:

The [gradle build](https://github.com/craftercms/studio/blob/1433f32c5fc326c6e63bd7463a8f3dd6ec5aeb40/pom.xml#L655) will execute this task. Essentially, adds exec permissions to the build.sh (i.e. `chmod +x build.sh`) and then runs bash script (i.e. `./build.sh`), which simply copies a bunch of files to the target location (see other comment).

Will this work in all relevant build execution contexts/scenarios?

Is there a better way to do this? It's important that the [npm install](https://github.com/craftercms/studio/blob/1433f32c5fc326c6e63bd7463a8f3dd6ec5aeb40/pom.xml#L645) task runs first, but once that completes, the bash needn't run via a yarn task if there's a better way.

I could do this via node apis instead of bash but they'd end up doing same thing.

Reply:
If you only need to copy files I would suggest doing it in maven instead of a new script, take a look at https://github.com/craftercms/studio/blob/develop/pom.xml#L747

Reply:
Besides the copying of files, the bash script also performs a file concatenation. See line 19 of the build.sh below.

Reply:
@jdrossl any ideas here or will we go with bash?

Reply:
maven can only copy files, not append to them